test_runner/performance/test_perf_olap.py

```diff
@@ -0,0 +1,89 @@
+from typing import Dict, List, Tuple
+
+import pytest
+from fixtures.compare_fixtures import RemoteCompare
+from fixtures.log_helper import log
+
+# A list of queries to benchmark from ClickBench.
+# The first element of each tuple is the query id to make it distinguishable in logs/grafana charts/etc.,
```

Comment:
what do you think about having a more structural view of this Tuple? `dataclass`, `namedtuple`, etc. Because I don't think that it will be cool enough to recall what do we store in each index